src/libs/tmdk/tests/deploy.test.ts

```diff
@@ -1,26 +1,265 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
 // Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 import { handler, Options } from "../src/commands/deploy";
 import { Arguments } from "yargs";
-import { mockExit } from "./test-utils";
+import {
+  workspaceId,
+  mockExit,
+  s3BucketArn,
+  s3ContentLocationBase,
+  workspaceBucket,
+} from "./test-utils";
+import { mockClient } from "aws-sdk-client-mock";
+import {
+  CreateComponentTypeCommand,
+  CreateEntityCommand,
+  CreateSceneCommand,
+  GetComponentTypeCommand,
+  GetEntityCommand,
+  GetWorkspaceCommand,
+  IoTTwinMakerClient,
+  ResourceNotFoundException,
+} from "@aws-sdk/client-iottwinmaker";
+import {
+  emptyTmdk,
+  componentType1Definition,
+  scene1,
+  entity1Definition,
+} from "./test-constants";
+import { PutObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import * as fsPromises from "fs/promises";
+const fs = require("fs");
 
-jest.mock("../src/lib/aws-clients");
+jest.mock("fs/promises", () => {
+  return { readFile: jest.fn() };
+});
+const twinmakerMock = mockClient(IoTTwinMakerClient);
+const s3Mock = mockClient(S3Client);
+const fakeTmdkDir = "/tmp/deploy-unit-tests";
 
 describe("testing deploy", () => {
```

Comment:
all tests are always testing, so no need to say that the test is testing

Reply:
Ok, for test suites how do we prefer to name them?

Reply:
Typically use `describe` block to logically group tests, for example, in this test suite: https://github.com/awslabs/iot-app-kit/blob/main/packages/core/src/common/dataFilters.spec.ts

each function being tested in a file that exports several, has a describe block.

For your use case, i would probably do something such as:


describe('deploys succesfully')
and

describe('fails to deploy')

as a way to categorize your test suites.

Since your deploy tests are contained in a deploy test file, it's already implied (and clear in the jest test runner results) that they are all related to deployment, so a single file level wide describe('deploys') adds no information though its a common pattern